From 2852e40e4326f22b81e4f53fde203a16893a9131 Mon Sep 17 00:00:00 2001
From: Mario Schuknecht <mario.schuknecht@dresearch-fe.de>
Date: Wed, 8 Jul 2020 07:11:19 +0200
Subject: [PATCH] ext4 64bit feature not activated by default HYP-24046

The ext4 64bit feature allows partitioning greater than 16TiB. But with
older u-boot versions it leads to boot problems, because the ext4 64bit
feature is not yet supported.
To remain backwards compatible, the ext4 64bit feature is no longer
activated by default.

Signed-off-by: Mario Schuknecht <mario.schuknecht@dresearch-fe.de>
---
 misc/mke2fs.conf.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/misc/mke2fs.conf.in b/misc/mke2fs.conf.in
index c95f55bb..e7972dea 100644
--- a/misc/mke2fs.conf.in
+++ b/misc/mke2fs.conf.in
@@ -11,8 +11,9 @@
 		features = has_journal
 	}
 	ext4 = {
-		features = has_journal,extent,huge_file,flex_bg,metadata_csum,64bit,dir_nlink,extra_isize
+		features = has_journal,extent,huge_file,flex_bg,metadata_csum,dir_nlink,extra_isize
 		inode_size = 256
+		auto_64-bit_support = 1
 	}
 	small = {
 		blocksize = 1024
-- 
2.16.4

